### ISSUE TITLE
Browser detection

### ISSUE THREAD

**Issue by Cowicide** [sophron/wifiphisher#393]
**Version**: lastest

**Distribution**: Kali Linux

**Summary**: If the Mac user is utilizing Safari, a Chrome page is shown.

**Description**: wifiphisher should detect the browser the user is utilizing and show the correct Internet error screen.  For example, for Safari users:

![screen shot 2017-01-05 at 4 38 43 pm](https://cloud.githubusercontent.com/assets/716801/21702081/ad145c42-d366-11e6-9042-341bbc7c8abb.png)

**Comment by laozi999** [comment]
You could trigger this using javascript and then redirecting the user, but I don't think wifiphisher lets you navigate outside the phishing-page folder that you choose when running wifiphisher.

Something like this:
```
if (/Windows phone|windows phone/i.test(navigator.userAgent)) {
    // Show page for Windows Phone
} else if (/Android|android/i.test(navigator.userAgent)) {
    // Show page for Android
} else if (/iPad|iPhone|iPod|ipad|iphone|ipod|ios|iOS/.test(navigator.userAgent) && !window.MSStream) {
    // Show page for iOS
} else { // if not mobile
    if (/Firefox|firefox/i.test(navigator.userAgent)) {
        // Show page for Firefox
    } else if (/Safari|safari/i.test(navigator.userAgent)) {
        // Show page for Safari
    } else if (/Opera|opera/i.test(navigator.userAgent)) {
        // Show page for Opera
    } else if (/Chrome|chrome/i.test(navigator.userAgent)) {
        // Show page for Chrome
}
```

You can of course rearrange these and have for example only Firefox browsers on Android devices display a certain page. Like so:

```
if (/Android|android/i.test(navigator.userAgent) && /Firefox|firefox/i.test(navigator.userAgent)) {
    // Show page for Firefox users on Android
}
```

**Comment by sophron** [comment]
Correct, we currently support only Chrome's page. We should put the pages from the rest of major browsers too.

**Comment by laozi999** [comment]
I think it's better to merge them all in one scenario, that way you can just keep updating 1 file rather than every individual scenario.

**Comment by sophron** [comment]
Yes, all of them are in one scenario. The idea is that we determine the victim's OS & browser based on the User-Agent header and craft the page accordingly.

**Comment by JustMev** [comment]
need to add ssl  OAuth Login Page  A free Wi-Fi Service asking for Facebook credentials to authenticate using OAuth, as this would be more realistic and not frighten people with ssl warning
